src/plugins/multiline/index.ts

```diff
@@ -0,0 +1,393 @@
+      const commaBreaker = {
+        symbol: ",",
+        minWidth,
+        mode: CollapseMode.Always,
+        indent: 0,
+      };
+      const equalsBreaker = {
+        symbol: "=",
+        minWidth,
+        mode: CollapseMode.Always,
+        indent: 20,
+      };
+      const arithmeticBreakers = ["+", "−", "·"].map((s) => ({
+        symbol: s,
+        minWidth,
+        mode: CollapseMode.AtMaxWidth,
+        indent: 20,
+      }));
+
+      // add line breaks
+      verticalify(
+        f,
+        {
+          enclosingBracketType: undefined,
+          containerType: "root",
+          domManipHandlers,
+        },
+        {
+          collapse: {
+            functionCall: { symbols: [commaBreaker] },
+            functionDef: { symbols: [] },
+            all: { symbols: [...arithmeticBreakers] },
+            root: { symbols: [equalsBreaker, commaBreaker] },
+            other: { symbols: [] },
+            list: {
+              symbols: [{ ...commaBreaker, mode: CollapseMode.AtMaxWidth }],
+            },
+            piecewise: { symbols: [commaBreaker] },
+          },
+          skipWidth: minWidth,
+          minPriority: 0,
+          maxPriority: 1,
+        }
+      );
+
+      // perform all dom writing (to prevent getBoundingClientRect-related slowdowns)
+      for (const h of domManipHandlers) h();
+    }
+
+    // clear multilinification cache
+    this.pendingMultilinifications = new Set();
+  }
+
+  keydownHandler = (e: KeyboardEvent) => {
+    if (e.key === "ArrowLeft" || e.key === "ArrowRight") {
+      const cursor = document.querySelector(".dcg-mq-cursor");
+      if (cursor) {
+        this.lastRememberedCursorX = cursor.getBoundingClientRect().left;
+      }
+    }
+
+    if (e.key.toUpperCase() === "M" && e.ctrlKey) {
+      this.dequeueAllMultilinifications();
+    }
+  };
+
+  mousedownHandler = () => {
+    // get the mathquill cursor position x
+    setTimeout(() => {
+      const cursor = document.querySelector(".dcg-mq-cursor");
+      if (cursor) {
+        this.lastRememberedCursorX = cursor.getBoundingClientRect().left;
+      }
+    });
+  };
+
+  resizeHandler = () => {
+    this.afterConfigChange();
+  };
+
+  dispatcherID: string | undefined;
+
+  customDispatcherID: number | undefined;
+
+  afterEnable() {
+    document.addEventListener("keydown", this.keydownHandler);
+    document.addEventListener("mousedown", this.mousedownHandler);
+    document.addEventListener("resize", this.resizeHandler);
+
+    this.afterConfigChange();
+
+    this.multilineIntervalID = setInterval(() => {
+      if (
+        Date.now() - this.lastEditTime <
+          this.settings.multilinifyDelayAfterEdit ||
+        !this.settings.automaticallyMultilinify
+      )
+        return;
+
+      this.dequeueAllMultilinifications();
+    }, 0);
+
+    this.dispatcherID = Calc.controller.dispatcher.register((e) => {
+      if (
+        e.type === "set-item-latex" ||
```

Comment:
Also check for `e.type === "ui/container-resized"`? So it updates when window is resized.

Reply:
Fixed!